certif/app/styles/pages/authenticated/team.scss

```
@@ -0,0 +1,41 @@
+.team {
+  &__no-referer-container {
+    margin-bottom: 24px;
+    padding: 6px;
+  }
+
+  &__no-referer-container-grey {
+    padding: 16px;
+    background: $pix-neutral-5;
+    display: flex;
+  }
+
+  &__no-referer-title {
+    font-weight: 500;
```

On peut regrouper un peu:
```suggestion
  &__no-referer
     &-container {
       margin-bottom: 24px;
       padding: 6px;
       
       &-grey {
        padding: 16px;
        background: $pix-neutral-5;
        display: flex;
      }
    }

   &-title {
    font-weight: 500;
```